suites/test_git_import.py

```diff
@@ -13,85 +13,101 @@
 git_import_repo_URL = base.config_reader('git_import_test_repo', 'git_import_repo_url')
 git_import_repo_Name = base.config_reader('git_import_test_repo', 'git_import_repo_name')
 git_import_repo_branch = base.config_reader('git_import_test_repo', 'git_import_repo_branch')
-number_of_modules = base.config_reader('git_import_test_repo', 'number_of_modules_imported')
-module_title_prefix = base.config_reader('git_import_test_repo', 'module_prefix')
-assembly_title_prefix = base.config_reader('git_import_test_repo', 'assembly_prefix')
-
+# number_of_modules = base.config_reader('git_import_test_repo', 'number_of_modules_imported')
+# module_title_prefix = base.config_reader('git_import_test_repo', 'module_prefix')
+# assembly_title_prefix = base.config_reader('git_import_test_repo', 'assembly_prefix')
+number_of_resources_uploaded = base.config_reader('git_import_test_repo', 'number_of_resources_uploaded')
+number_of_modules_uploaded = base.config_reader('git_import_test_repo', 'number_of_modules_uploaded')
+number_of_assemblies_uploaded = base.config_reader('git_import_test_repo', 'number_of_assemblies_uploaded')
 
 
 @lcc.suite(description="Suite: Git import functionality", rank=6)
 class test_git_import:
-  api_auth = lcc.inject_fixture("api_auth")
-
-  @lcc.disabled()
```

Comment:
@pbadak I see you removed the existing test, why is that the case? This can be one e2e check we need, could be the last one if that be the case, no?

Reply:
As we are verifying all git2pantheon APIs in this test , existing test validation is also getting covered so there is no need to have it seperately. cc : @noopurmath3